src/Microsoft.Extensions.DotnetToolDispatcher.Sources/project.json

```diff
@@ -2,7 +2,7 @@
   "version": "1.2.0-*",
   "shared": "*.cs",
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview2-003121",
```

Comment:
Depends. Does this need project.json support? If so, preview2-1. Otherwise, preview3 is the latest.